fail2ban/tests/files/logs/postfix-address

```diff
@@ -0,0 +1,2 @@
+# failJSON: { "time": "2016-07-15T17:40:29", "match": true , "host": "1.1.1.1" }
```

Comment:
All our test cases are running in year 2005, that's why the test cases fail... Please correct both times in failJSON.